index.rst

```diff
@@ -45,12 +42,53 @@ All of this data needs to be associated with this configuration in a way that is
 In this tech-note we discuss the basic organization of configurations and the operational procedures involved in handling CSC configuration.
 The document begins with the most simple case of bringing a CSC up with a specific configuration, and eventually details how to updating and testing newly defined configurations.
 
-Most CSCs handle configuration in the same manner, however, there are a few exception to the rule, notably the Camera CSCs.
-These special cases are addressed in section-appendix-configuration-non-salObj_.
 
-For more technical details about how CSCs handle configuration, including the definition of configuration related topics published by the CSC, see `tstn-017 <https://tstn-017.lsst.io>`__.
+Configuration Interface Summary
+===============================
+
+The standard interface for CSCs is defined in :lse:`209` and is the official reference/requirements document.
+This section provides a very brief overview of the key topics related to configuration that are continually mentioned in the sections below.
+
+The standard configuration related interactions with CSCs happens via two main events (``configurationsAvailable`` and ``configurationsApplied``) and a single parameter as part of the start command (``configurationOverride``).
+When in the STANDBY state, the ``configurationsAvailable`` event is published which a list of non-default (custom) configurations as well as information required to track available configurations (both default and custom) to a specific git commit.
+Under normal operations, site-specific default configurations are automatically loaded.
+However, using the ``configurationOverride`` parameter (explained in detail in the use-cases below), it is possible to specify one of the non-default configurations that is listed in the ``configurationsAvailable`` event.
```

Comment:
I gently suggest that "(explained in detail in the use-cases below)" is implied and should be removed. This whole section is a brief overview. Or if you have a link to more information, you could use that; it would be a more useful reference than this vague statement.